docs/run_publish/references.md

```diff
@@ -109,6 +43,10 @@ This flag allows you to set the batch size in the command line. If batch size is
 2021-08-09T23:24:49.235Z <fetch> INFO fetch block [6661,6680], total 20 blocks
 ```
 
+### --block-confirmations (Ethereum only)
```

Comment:
@stwiname I assume this parameter is independent to --unfinalized-blocks?. What is the default value for this?

Reply:
Default is 20. And it's independent and only used with unfinalized blocks